### PR TITLE
improvement: ZENKO-4546 storage consumption metrics bump PR

### PR DESCRIPTION
**Pull request by XinLi0207**
**What does this PR do, and why do we need it?**

**Which issue does this PR fix?**

fixes #<ISSUE>

**Special notes for your reviewers**:

### CODE REVIEW

File: solution/deps.yaml
Comment (XinLi0207) [inline solution/deps.yaml]: will change here post release zkop

**Review by KillianG — APPROVED** [review]

File: solution/deps.yaml
Comment (ghost) [reply solution/deps.yaml]: cc @francoisferrand as we have [ongoing discussions](https://scality.slack.com/archives/C031XT2FDA9/p1681830380769799) on this, maybe bumping ZKOP in dev/2.6 should be avoided, can you confirm?

Comment (XinLi0207) [reply solution/deps.yaml]: thanks for letting me know the concern about zkop in zenko 2.6, if don't bump zkop, can we introduce[ this env](https://github.com/scality/zenko-operator/pull/373/files#diff-e6542f41494ebe04c99b56959487c4a3fcec511efddb016773806f2da7896b66R143) for pensieve-api in Zenko instead of zkop?

Comment (ghost) [reply solution/deps.yaml]: I don't think so... Let's wait for a confirmation first, as we'll need a solution anyway to unblock this branch... Maybe it is fine for this bump, but we need confirmation :smile:

Comment (francoisferrand) [reply solution/deps.yaml]: ZKOP cannot be bumped in zenko 2.6 (yet), we have a ticket for this : https://scality.atlassian.net/browse/ZKOP-248
But it will probably be done next week.

Generally, if possible i think it is best to wait until CNES release (e.g. artesca 1.7.5) is complete to merge anything in zenko 2.6, to avoid adding more stuff to that release while in RC.

**Review by ghost — CHANGES_REQUESTED** [review]
Putting a lock here till we fix the 2.6 branch (should be fine after the next Aretsca 1.7.5 release)

**Review by ankit-gautam23 — COMMENTED** [review]
Is it bumped??

**Review by ghost — APPROVED** [review]

**Comment by XinLi0207** [comment]
ping

**Pull request merged**
